docs/development/python/manage-python-environments-pipenv.md

```diff
@@ -0,0 +1,162 @@
+---
+author:
+  name: Jared Kobos
+  email: docs@linode.com
+keywords: ["python", "virtual environment", "pip","virtualenv","pipenv"]
+description: 'Pipenv combines the functionality of Pip and Virtualenv into a single tool, helping to simplify workflows as you install packages and manage virtual environments. This guide will show you how to install Pipenv, create and work with virtual envrionments, and install packages.'
+og_description: 'Pipenv combines the functionality of Pip and Virtualenv into a single tool, helping to simplify workflows as you install packages and manage virtual environments. This guide will show you how to install Pipenv, create and work with virtual envrionments, and install packages.'
+license: '[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0)'
+modified: 2018-03-27
+modified_by:
+  name: Linode
+published: 2018-03-27
+title: Manage Python Packages and Virtual Environments with Pipenv
+---
+
+## What is Pipenv?
+
+[Pipenv](https://docs.pipenv.org/), Python's officially recommended package management tool, combines the functionality of Pip and Virtualenv, along with the best features of packaging tools from other languages such as Bundler and NPM. This results in a simplified workflow for installing packages and managing virtual environments.
+
+### Install Pipenv
+
+You will need to have Python installed on your system before installing Pipenv.
+
+If you are using Ubuntu 17.10 or higher, you can install Pipenv directly from the Pypa ppa:
+
+    sudo apt install software-properties-common python-software-properties
+    sudo add-apt-repository ppa:pypa/ppa
+    sudo apt update
+    sudo apt install pipenv
+
+Other distributions should first install Pip and use it to install Pipenv:
+
+    sudo apt install python-pip
+    pip install pipenv
+
+## Use Pipenv
+
+1.  Create a directory for an example Python project:
+
+        mkdir python-example && cd python-example
+
+2.  Create a virtual environment in the directory:
+
+        pipenv --python 3.6
+
+    {{< output >}}
+Creating a virtualenv for this project…
+Using /home/username/miniconda3/bin/python3.6m (3.6.4) to create virtualenv…
```

Comment:
We could hold this until end of April so the install on Py3 doesn't have to rely on Miniconda